### PR TITLE
dist/template: fix YAML indentation

### PR DESCRIPTION
**Pull request by lucab**
This fixes several YAML sections that are over or under indented.
It contains no functional changes.

/cc @steveeJ

### CODE REVIEW

**Comment by steveej** [comment]
Which tool did you use to autoformat this?

**Comment by lucab** [comment]
@steveeJ nothing for now. I just spot a couple of sections off, but the few linters/formatters I tried were making a larger mess, so I just adjusted them manually. I'll report back if/when I find something that I like, to hook in CI.

**Comment by steveej** [comment]
> I'll report back if/when I find something that I like, to hook in CI.

Thanks, that sounds like a reasonable goal. Let's keep it open until then.

**Comment by lucab** [comment]
I have added a linting script and fixed up all errors, PTAL. @steveeJ can you try if the script works for you? If so, once this lands I'll wire it up as a prow job.

File: dist/prow_yaml_lint.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+YAML_LINTER='yamllint'
+YAML_RULES='{extends: default, rules: {line-length: {max: 120}}}'
+YAML_LINT_CMD=("${YAML_LINTER}" '-s' '-d' "${YAML_RULES}")
+
+if ! type -f "${YAML_LINTER}"; then
+  echo "error: could not find ${YAML_LINTER} in PATH"
+  exit 1
+fi
+
+find . -type f -name '*.yaml' -print0 | xargs -0 "${YAML_LINT_CMD[@]}"
```
Comment (steveej) [inline dist/prow_yaml_lint.sh]: I think we could also match `*.yml` here

**Review by steveej — COMMENTED** [review]
Confirmed working here.
I added a suggestion and I'll leave it to you to dismiss that.

/hold
/lgtm

**Comment by lucab** [comment]
I've added the `.yml` portion too, but the green-stamp is now gone. It needs another round of review.

**Pull request merged**